docker/build_artifacts/sagemaker/tfs_utils.py

```diff
@@ -26,11 +26,12 @@
 CUSTOM_ATTRIBUTES_HEADER = "X-Amzn-SageMaker-Custom-Attributes"
 
 Context = namedtuple("Context",
-                     "model_name, model_version, method, rest_uri, grpc_port, "
+                     "model_name, model_version, method, rest_uri, grpc_port, channel, "
                      "custom_attributes, request_content_type, accept_header, content_length")
 
 
-def parse_request(req, rest_port, grpc_port, default_model_name, model_name=None):
+
+def parse_request(req, rest_port, grpc_port, default_model_name, model_name=None, channel=None):
```

Comment:
can we can place the "channel" arg before "model_name" arg and remove the default None value? I think channel object is always passed to this method.

Reply:
This method is also called [here](https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/docker/build_artifacts/sagemaker/python_service.py#L240), where channel is not passed in.
So we need to specify the passed-in channel/model arg while calling this method.

Reply:
I see. make sense.